client/layout/masterbar/blaze-pro.scss

```diff
@@ -1,7 +1,21 @@
 @import "@wordpress/base-styles/breakpoints";
+@import "@automattic/components/src/styles/typography.scss";
+
+@font-face {
+	font-family: InterVariable;
+	font-style: normal;
+	font-weight: 400 700;
+	font-display: swap;
+	src: url(../../assets/fonts/Inter-VariableFont.ttf) format("truetype");
+	src: url(../../assets/fonts/Inter-VariableFont.woff2) format("woff2");
```

Comment:
Inter font should have been loaded from
wp.com (see [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/videopress-step-wrapper/style.scss#L1-L10)). Can we use it instead of adding the font to repo? 🙂

Reply:
Thanks @chihsuan , applied it in https://github.com/Automattic/wp-calypso/pull/91637/commits/f4213d92bd1643d5e74e36f64e6e954286387104.